main/src/library/Chain.flix

```diff
@@ -759,7 +759,7 @@ namespace Chain {
         case Some(x) =>
             let arr = Array.new(x, length(c));
             let f = (i, b) -> { arr[i] = b; i + 1 };
-            foldLeft(f, 0, c);
+            let _ = foldLeft(f, 0, c);
```

Comment:
I think this shows that the programmer should use foreach, right?

Reply:
yes, but this is where zipWithIndex was wanted, so this was the alternative.